screenpipe-server/src/migrations/20240716120000_add_fields_to_ocr_text.sql

```diff
@@ -0,0 +1,13 @@
+-- Add new columns to the ocr_text table
```

Comment:
first thought is, what if we stop using tesseract? tesseract is very old ocr engine which uses old models and tons of hacks which makes it quite a pain in our workflow atm